index.js

```diff
@@ -6,12 +6,39 @@ module.exports = {
   extends: ['airbnb'],
   plugins: ['chai-friendly'],
   rules: {
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
+    'no-unused-expressions': 'off',
+    'chai-friendly/no-unused-expressions': 'error',
     'linebreak-style': ['error', 'unix'],
     'no-param-reassign': ['error', { props: false }],
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
-    'consistent-return': 0,
+    'no-unused-vars': ['error', {'vars': 'all', 'args': 'none'}],
+    'consistent-return': 'off',
+    'arrow-body-style': ['warn', 'as-needed', {
+      requireReturnForObjectLiteral: false,
+    }],
+    'prefer-rest-params': 'warn',
```

Comment:
i dont see the point of not inforcing rest perams when you just do `const args = arguments`